### DISCUSSION TITLE
can you change the colours for the floating instance?

### DISCUSSION THREAD

**Discussion by tokisuno** [nvim-tree/nvim-tree.lua#2319] [Q&A] [answered]
![image](https://github.com/nvim-tree/nvim-tree.lua/assets/85533116/51897ea1-c822-4b09-9364-7c5486f12d34)

not a fan of these colours. any way to change them?

**Comment by alex-courtis** [comment, marked as answer]
You can change the colours for the floating window in the same manner as the regular window: `:help nvim-tree-highlight`

You can change the colours for the floating window itself via `:help nvim-tree.view.float.open_win_config`
